hedera-node/src/main/java/com/hedera/services/context/properties/GlobalDynamicProperties.java

```diff
@@ -275,6 +276,8 @@ public void reload() {
         requireMinStakeToReward = properties.getBooleanProperty("staking.requireMinStakeToReward");
         nodeMaxMinStakeRatios =
                 properties.getNodeStakeRatiosProperty("staking.nodeMaxToMinStakeRatios");
+        maxInternalContractCreations =
```

Comment:
What do you think about,
```
maxInternalContractCreations = Math.min(
    maxFollowingRecords,
    properties.getIntProperty("contracts.maxInternalContractCreations"));
```
so that in the tests below we can just do `numAllocatedIds < dynamicProperties.maxInternalContractCreations()`?

Reply:
makes sense. resolved.